### DISCUSSION TITLE
Can I create a custom ECM model?

### DISCUSSION THREAD

**Discussion by MarkVeerasingam** [pybamm-team/PyBaMM#4573] [Q&A]
Hi folks,
Im looking at expanding my learnigns with pybamm. I am complete noob to battery modelling and want to get started on ECMs.
I know pybamm offer a 1RC and 2RC Thevenin model. However is it possible to use pybamms Base Model class to create something simple like a Rint model or higher RC Order in the Thevenin model?

**Comment by valentinsulzer** [comment]
Yes, you just need to define the ODEs explicitly yourself - see https://docs.pybamm.org/en/stable/source/examples/notebooks/creating_models/1-an-ode-model.html

**Reply by valentinsulzer** [reply]
Another example is https://github.com/pybamm-team/PyBaMM/blob/f05cae2ecda531cde049ccfc69d470cdb845bf98/src/pybamm/models/full_battery_models/lithium_ion/basic_splitOCVR.py#L7
